pyrobosim/pyrobosim/navigation/planner_base.py

```diff
@@ -1,147 +0,0 @@
-""" Specification of the interface that all planners must implement. """
-
-import matplotlib.pyplot as plt
-from matplotlib.collections import LineCollection
-
-from pyrobosim.utils.motion import Path
-
-
-class PathPlannerBase:
-    """The base class for path planners."""
-
-    def __init__(self):
-        """
-        Creates an instance of PathPlannerBase.
-        """
-        self.goal = None
-        self.start = None
-        self.impl = None
-        self.planning_time = 0.0
-        self.graphs = []
-        self.graphs_updated = False
-        self.latest_path = Path()
-
-    def reset(self):
-        """
-        Resets the state of the planner.
-        Sub-classes should add their own relevant mechanisms
-        """
-        self.goal = None
-        self.start = None
-        self.planner = None
-        self.planning_time = 0.0
-        self.graphs = []
-        self.latest_path = Path()
-        self.graphs_updated = False
-
-    def plan(self, start, goal):
-        """
-        Plans a path from start to goal.
-
-        :param start: Start pose or graph node.
-        :type start: :class:`pyrobosim.utils.pose.Pose` /
-            :class:`pyrobosim.utils.search_graph.Node`
-        :param goal: Goal pose or graph node.
-        :type goal: :class:`pyrobosim.utils.pose.Pose` /
-            :class:`pyrobosim.utils.search_graph.Node`
-        :return: Path from start to goal.
-        :rtype: :class:`pyrobosim.utils.motion.Path`
-        """
-
-        raise NotImplementedError(
-            f"Subclasses must implement {self.__class__.__name__}.plan()"
-        )
-
-    def info(self):
```

Comment:
There are still a few references to this function lying around up in the examples that you should remove, I don't think you need them anyway... 

E.g. https://github.com/sea-bass/pyrobosim/pull/241/files#diff-1205b2ba174317521441667503ceb5cfe01fa4386037f35800e7d8d65504be21R32

Reply:
Oh yeah, nice! Replaced with `pyrobosim.utils.motion.Path.print_details()`